srtcore/core.cpp

```diff
@@ -210,8 +210,9 @@ struct SrtOptionAction
         // passed to a setting function.
         private_default[SRTO_STREAMID] = string();
     }
-}
-srt_options_action;
+};
+
+static const SrtOptionAction s_srt_option_action;
```

Comment:
In C++ const is static by default.

Reply:
`s_` is not exactly right prefix for a global constant - this is for static class members.

Reply:
Changed to `g_` and moved under `srt` namespace.